Expanded and clarified explanation of what's happening in the final s… (#397)
Agoric/documentation: main/getting-started/before-using-agoric.md

      <td>9</td>
      <td>Install the Agoric CLI by: <code>yarn link-cli 
        &lt;<i>agoric script location</i>&gt;</code></td>

>       <td>Select a location for the Agoric CLI program to be called from. For example,

Reviewer: ```suggestion
      <td>Select a location for the Agoric CLI wrapper script. For example,
```

        <code>yarn link-cli /usr/local/bin/agoric</code> (or if that fails
        with permission problems, <code>sudo yarn link-cli /usr/local/bin/agoric</code>)

>         This creates a shell script named <code>agoric</code> in the specified directory.
>         The script runs the <code>agoric</code> commands that make up the Agoric CLI.

Reviewer: ```suggestion
        This creates a <code>/usr/local/bin/agoric</code> shell script that you can run to access the SDK's Agoric CLI.
```

Author: I'm a little bit confused about the correction. In the example case established on lines 94-95, yes it's creating /usr/local/bin/agoric. But my understanding is that it'll create the agoric file in whatever directory is specified in the command, thus my current phrasing above in line 96. If it's always created in /usr/local/bin/agoric, I need to also change something in the previous column's text. Also, is it "can run"? If this is what the script does, I'd prefer the more definitive "shell script that when run accesses the.."

Reviewer: It doesn't have to be called "agoric".  If you say `/usr/local/bin/fororo`, then it will create a file called "fororo".

If you prefer "shell script that when run..." we can leave it that way.

Author: How about "This creates a shell script that when run accesses the SDK's Agoric CLI. We strongly recommend putting it at <code>/usr/local/bin/agoric</code>, but its specific location and name is up to you."    Which covers both a possible different directory than /usr/local/bin and a possible different name, but gives strong guidance as to a best practice for it.

Reviewer: I like it!
